src/main/resources/templates/home.html

```diff
@@ -15,7 +15,7 @@
         </tr>
         </thead>
         <tbody>
-        <tr th:each="entry : ${entries}">
+        <tr th:each="entry : ${entries}" th:if="${entry != null}">
```

Comment:
Can this still happen in the new scenario?

Reply:
Good point, removed...

Reply:
I was wondering, it might be a nice safeguard to prevent the UI from failing in front of an end-user. But if we can guarantee this never happens from the back-end, it wouldn't be necessary.